src/index.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
```

Comment:
Since you're using the latest version of react, you don't need to import React at all, because React does it for you by default.

Reply:
Hi @Ntwali-Josue  the reason behind still importing react from react on that file I used react strict mode, and it can't work without importing react.